### ISSUE TITLE
Redesign conversations

### ISSUE THREAD

**Issue by a-rasin** [iNPUTmice/Conversations#4189]
The current design feels too old, you can get used to it but imo a more "modern" design would be better.
That's what a good amount of users have been saying in the Conversations room, the responses they would get was "What do you mean exactly, what needs to change?", that is the reason of this issue, hopefully it clears things a bit more.
![image](https://user-images.githubusercontent.com/90502898/135604526-5c7bae9d-ba9a-4db5-b8d5-6c1a46f69ef7.png)

Something like this would be nice.

Concretely:
Rounded message bubbles.
Smaller rounded avatars.
Nick of user above a group of bubbles and not inside each bubble.
Reduced space between grouped bubbles.
The color of the title bar should be the same color as the background maybe?
Be able to change the main color.
Use rounded avatars everywhere.

Threaded replies and reactions would also be pretty cool but those probably need another issue.

**Comment by syntafin** [comment]
When an update of the design, please like at the moment adept the Material You design.

But I think, it's easier to get these, if someone with experience in this can create a PR for it.

**Comment by ghjardim** [comment]
I agree that Conversations should have a redesign. Possibly taking Telegram as inspiration, and allowing customization by the user, so that you can leave the app looking like it is now, or in a more "modern" look.

**Comment by syntafin** [comment]
I don't know how Telegram looks (I don't use it), but would it not be better if there would be a redesign, if switchable or not, to take the step to Material You?

**Comment by ghjardim** [comment]
@syntafin About Material You, I definitely think that it is an option.
Explaining about Telegram, it helps you define which colors the app uses (mainly, the colors for your messages, the received messages, and an "accent color" for other things such as icons and replyed messages). Telegram also allows you to define the background image and how rounded the corners of the message bubbles are.
It can help if you search for the terms "telegram themes".
An idea for the future: Conversations could have multiple themes, such as Material You, Dark, OLED; also green, blue, yellow... All customizable by the user.

**Comment by nvidiaLinuxUser** [comment]
I love Conversations, the only thing my friends and I didn't like was the design. Conversations would look good with Material You.

**Comment by S0ulf3re** [comment]
I would also love to see Material You and a bit of the Material 3 design language as well.
